pkgpanda/__init__.py

```diff
@@ -792,10 +797,10 @@ def _get_service_names(_dir):
             if package.username is not None:
                 sysusers.add_user(package.username, package.group)
 
-            # Ensure the state directory in `/var/lib/dcos` exists
+            # Ensure the state directory exists
             # TODO(cmaloney): On upgrade take a snapshot?
             if self.__manage_state_dir:
-                state_dir_path = '/var/lib/dcos/{}'.format(package.name)
+                state_dir_path = self.__state_dir_root + '/' + package.name
```

Comment:
you might want to consider using
```
state_dir_path = os.path.join(self.__state_dir_root, package.name)
```
And perhaps use normpath, too:
```
state_dir_path = os.path.normpath(os.path.join(self.__state_dir_root, package.name))
```
That way it's a bit more robust against extra slashes, etc.